src/lib.rs

```diff
@@ -23,7 +23,7 @@ mod window_identifier;
 pub use self::window_identifier::WindowIdentifier;
 mod app_id;
 pub use self::app_id::AppID;
-mod file_name;
+mod file_path;
```

Comment:
This also needs a `pub use self::file_path::FilePath;` and also please squash the commits that adds the type and the one that renames it